Sources/StreamChatSwiftUI/ChatChannel/ChatChannelView.swift

```diff
@@ -196,6 +201,9 @@ public struct ChatChannelView<Factory: ViewFactory>: View, KeyboardReadable {
         .alertBanner(isPresented: $viewModel.showAlertBanner)
         .accessibilityElement(children: .contain)
         .accessibilityIdentifier("ChatChannelView")
+        .if(utils.messageListConfig.bouncedMessagesAlertActionsEnabled) {
```

Comment:
maybe we just apply the modifier here, while move the if-check in the modifier itself?

Reply:
In fact, we can even ignore the config, since this will only be triggered when we call `showBouncedMessagesActionView()`, and we already use the flag there 🤔

Reply:
Done